command.go

```diff
@@ -138,6 +142,15 @@ func ExecuteHide(c Command, v *VHS) {
 	v.PauseRecording()
 }
 
+func ExecuteRequire(c Command, v *VHS) {
+	for _, t := range strings.Split(c.Args, ",") {
+		if _, err := exec.LookPath(t); err != nil {
+			log.SetFlags(0)
+			log.Fatalf("The VHS tape requires the following binaries:\n%s\n", c.Args)
```

Comment:
I think it would be bit more idiomatic to do:

```suggestion
			fmt.Printf("The VHS tape requires the following binaries:\n%s\n", c.Args)
			os.Exit(1)
```

Reply:
also, probably worth mentioning which of the required binaries is missing

Reply:
Thanks for the review.

> ```go
> fmt.Printf("The VHS tape requires the following binaries:\n%s\n", c.Args)
> ```

I mainly chose `log.Fatalf` because it sends to `os.Stderr` as well as `exit`s with 1.

Should I go with:

```go
fmt.Fprintf(os.Stderr, "The VHS tape requires the following binaries:\n%s\n", c.Args)
os.Exit(1)
```

> worth mentioning which of the required binaries is missing

That's a good idea! Will implement.